ember-flight-icons/tests/dummy/app/templates/engineering.hbs

```diff
@@ -1,34 +1,38 @@
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+<p>There are multiple ways to use these icons in your codebase. The <a href="#use-other" class="ds-a">package itself can be consumed</a> for direct import, or it can be installed as an <a href="#ember-flight-icons" class="ds-a">Ember addon</a> for the convenience of using a component with strong defaults.</p>
+</div>
 
 <div class="ds-info">
-<p class="ds-p">🚨 Note: All of the flight-icon npm packages are currently in beta and any production usage should be discussed with the Design Systems team first.</p>
+<p >🚨 Note: All of the flight-icon npm packages are currently in beta and any production usage should be discussed with the Design Systems team first.</p>
 </div>
 
-<h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons">&sect;</a> Use in Ember Apps</h2>
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+<h2 class="ds-h2" id="ember-flight-icons"><a href="#ember-flight-icons" class="ds-a" rel="external">&sect;</a> Use in Ember Apps</h2>
 <h3 class="ds-h3">Installation</h3>
-<p class="ds-p">To install, run
+<p >To install, run
 <code class="ds-code">yarn add @hashicorp/ember-flight-icons</code>
 </p>
+</div>
+<div class="mt-8 prose prose-lg text-gray-800 pb-6">
+<h3 class="ds-h3" id="understanding"><a href="#understanding" class="ds-a" rel="external">&sect;</a> Understanding the component</h3>
 
-<h3 class="ds-h3" id="understanding"><a href="#understanding">&sect;</a> Understanding the component</h3>
-
-<p class="ds-p">The component comes with the following defaults:
-<ul class="ds-ul">
+<p >The component comes with the following defaults:
+<ol>
   <li class="ds-li"><kbd class="ds-kbd">fill</kbd> attribute: set to currentColor</li>
   <li class="ds-li"><kbd class="ds-kbd">id</kbd> attribute: a unique, automatically generated id</li>
   <li class="ds-li"><kbd class="ds-kbd">aria-hidden</kbd> attribute: set to true</li>
   <li class="ds-li"><kbd class="ds-kbd">height</kbd> and <kbd class="ds-kbd">width</kbd>: default size of 16x16 (px)</li>
   <li class="ds-li">(CSS)<kbd class="ds-kbd">class</kbd>: flight-icon, flight-icon-NAME, flight-icon-display-inline</li>
   <li class="ds-li">CSS display: set to <kbd class="ds-kbd">display:inline-block</kbd></li>
   <li class="ds-li"><kbd class="ds-kbd">data-test-icon</kbd> attribute: for the author's testing convenience</li>
-</ul>
+</ol>
 </p>
 
-<p class="ds-p">This makes the base, required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
 
-<br>So this invocation:
+<p >This makes the base, required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
```

Comment:
```suggestion
<p>This makes the base, required invocation quite terse- <kbd class="ds-kbd">@name</kbd> is the only property that requires specification.
```

suggestion (non-blocking): there are a few of these, we could add linting down the road? maybe a bad copy pasta if want to fix here

suggestion (non-blocking): would be cool to get rid of the "`" displaying in `<code>` and `<pre>`

happy to help with either task if you want a hand!

Reply:
> suggestion (non-blocking): would be cool to get rid of the "`" displaying in code and pre tags 

@amyrlam could you explain what you mean by this one?

Reply:
also, I'll run a formatter on the primary branch and update everything at once 👍 Thanks for catching these unsightly details.

Reply:
right now the back tick literally displays:

![image](https://user-images.githubusercontent.com/1372946/133126112-2e6fd4fc-5387-43e2-93dd-cbc759efe3af.png)

![image](https://user-images.githubusercontent.com/1372946/133126170-66273653-04e9-4b6e-b7ee-8af38252bc7b.png)

//

as for formatting, sounds good, would be great to have it done in CI separately (not blocking this branch) down the road